lib/shelley/src/Cardano/Wallet/Shelley.hs

```diff
@@ -288,7 +290,7 @@ serveWallet
                     Server.idleWorker
                 shelleyApi <- apiLayer (newTransactionLayer net) nl
                     (Server.manageRewardBalance proxy)
-                multisigApi <- apiLayer (newTransactionLayer net) nl
+                multisigApi <- apiLayer undefined nl
```

Comment:
Whoops? I understand you want to do this in a second PR and there's currently nothing wired into the transaction layer but may we use a more descriptive error than `undefined` here :grimacing: ?

Reply:
done